### PR TITLE
ACA-1303: Fix description link in provision_vm role README file

### PR DESCRIPTION
**Pull request by bardielle**
Updated the provision_vm_guest_id description to point to vSphere 7.0

### CODE REVIEW

File: roles/provision_vm/README.md [hunk 1]
```diff
@@ -406,7 +406,7 @@ N/A
     Set the guest ID.
     This parameter is case sensitive.
     This field is required when creating a virtual machine, not required when creating from the template.
-    Valid values are referenced here: https://code.vmware.com/apis/358/doc/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html
+    Valid values are referenced here: https://developer.vmware.com/apis/202/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html
```
Comment (machacekondra) [inline roles/provision_vm/README.md:409]: There is same issue also on line 559, can we fix it as well?

Comment (bardielle) [reply roles/provision_vm/README.md]: Fixed it as well, used that page https://developer.vmware.com/apis/1720/vsphere/    (vSphere Web Services API 8.0U2)

File: roles/provision_vm/README.md [hunk 2]
```diff
@@ -406,7 +406,7 @@ N/A
     Set the guest ID.
     This parameter is case sensitive.
     This field is required when creating a virtual machine, not required when creating from the template.
-    Valid values are referenced here: https://code.vmware.com/apis/358/doc/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html
+    Valid values are referenced here: https://vdc-download.vmware.com/vmwb-repository/dcr-public/184bb3ba-6fa8-4574-a767-d0c96e2a38f4/ba9422ef-405c-47dd-8553-e11b619185b2/SDK/vsphere-ws/docs/ReferenceGuide/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html
```
Comment (machacekondra) [inline roles/provision_vm/README.md:409]: As the link is quite long, maybe adding hyperlink like [here](https:...) would be better

**Review by machacekondra — APPROVED** [review]

File: roles/provision_vm/README.md (hunk 2 above)
Comment (bardielle) [reply roles/provision_vm/README.md]: Fixed it thanks.

**Pull request merged**